Homework_Weak1_Day_4.txt

```diff
@@ -0,0 +1,201 @@
+
+    var result = arr[i] * 3;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 3 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 4;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 4 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 5;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 5 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 6;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 6 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 7;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 7 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 8;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 8 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 9;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 9 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 10;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 10 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+}
+
+
+
+
+
+
+
+
+
+
+
+
+   --------------------------------------
+   problem  2 :
+
+function Grade (a){
+
+
+    for (var i = 60 ; i <= 100 ; i ++){
+      if (i >= 60 && i <= 69){
+       if (i === a) {
+           console.log (" For " + a + " you got D"); } } 
+
+
+   if (i >= 70 && i <= 79){
+        if (i === a) {
+           console.log(" For " + a + " you got C"); }}
+
+  if (i >= 80 && i <= 89){
+    if (i === a){
+      console.log(" For " + a + " you got B") }}
+
+if (i >= 90 && i <= 100){
+    if (i === a){
+      console.log(" For " + a + " you got A") }}
+
+}}
+
+-----------------------------------------
+
+
+
+
+
+
+
+
+
+problem 4 :
+ 
+ # Driving Age
+
+function drivingAge (age){
+
+    for ( var i = 0 ; i<= 100 ; i++){
+      
+
+       if (i >= 18){
+          if (i === age){
+        console.log("Drive Away!!") }}
+
+      else if (i < 18 ) {
+           if (i === age){
+        var calAge = 18 - i ;
+          console.log("Sorry you can not drive at this time, you have wait " +calAge+ " Years to start driving !!" ) } }
+
+
+} 
+}
+
+
+# what number is bigger ?
+
+
+var max = [10,3];
+
+
+for (var i = 0 ; i <= max.length ; i++) {
+   if (max[0] > max[1]) {
+     console.log("Number one is bigger")}
```

Comment:
Nice, and you can also say `console.log(max[0] + " is bigger")` to make it more user friendly